AsmSpy/ILogger.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace AsmSpy
+{
+    public interface ILogger
```

Comment:
Since we have only 1 implementation of this, do we really need an interface? I understand it might help in future but don't really see a need right now. Thoughts?